clients-src/modules/ROOT/pages/clients.adoc

```diff
@@ -6,125 +6,76 @@
 
 == Overview
 
-A TypeDB Client is any software that can connect to a TypeDB.
-It is meant to be used for the purpose of managing databases and performing operations (querying) with a
-database schema and data on the TypeDB server.
+Using a TypeDB Client is the primary way to interact with a TypeDB server. These include standalone applications
+as well as lower-level libraries called Drivers.
 
-TypeDB uses its own https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank] protocol via `gRPC`
-interface to accept connections from Clients. To connect we can use xref:studio.adoc[TypeDB Studio],
-xref:console.adoc[TypeDB Console], or we can build our own application (Client).
-
-A TypeDB Client uses `gRPC` to connect with a TypeDB server and provides more user-friendly interface: API, GUI or CLI.
-
-To build an application capable of connecting to TypeDB via *TypeDB Client RPC* protocol we can use one of the existing
-libraries implementing it, called <<_typedb_drivers,TypeDB Drivers>>, or we can implement it ourselves as a
-xref:new-driver.adoc[new driver].
+All Clients use a network `grpc` https://github.com/vaticle/typedb-protocol[protocol,window=_blank] under the hood
+to communicate with a TypeDB server.
 
 image::client-server-comms.png[Client types and server interaction]
 
 //#todo Update terminology on image
 
-Here is a list of existing *TypeDB Clients*:
-////
```

Comment:
Not sure if we needed this table, but wasn't being used so I removed it for now. We can always get it back via Git.